### ISSUE TITLE
Login screen default server

### ISSUE THREAD

**Issue by trapperhoney** [roundcube/roundcubemail#8640]
Hello,
I just upgraded to version 1.6.0 and I can't seem to get the login screen to use a default server.  Instead the login screen contains an empty 'Server' field.  Login works fine when I include my imap server in the Server field, but previously the 'default_host' configuration option automatically populated this box.

I see the release notes say that 'default_host' has been changed to 'imap_host'.  Is it possible that this migration broke the default login server feature?

I've tried using both 'imap_host' and 'default_host', but neither seem to work.  (The migration from 'smtp_server' to 'smtp_host' seems to have worked as expected).

**Comment by alecpl** [comment]
What do you have in `imap_host`? And what did you have in `default_host`?

**Comment by trapperhoney** [comment]
In both cases I was using 'tls://imap.domain.com'. I also tried the value from the defaults config ('localhost:143'). With the same results.

**Comment by alecpl** [comment]
With 'tls://imap.domain.com' the Server field should never appear, so something's off here.

**Comment by alecpl** [comment]
The Server field is shown if the imap_host is empty/unset or is an array with more than one element. So, please check your config again.

**Comment by trapperhoney** [comment]
I've stripped my config.inc.php down to a bare minimum, but even with this, I still see the Server box on the login screen.  Reverting back to 1.5.3 and changing imap_host to default_host hides the Server box, as expected.

```
<?php

$config['imap_host'] = 'tls://localhost';
$config['db_dsnw'] = 'mysql://roundcube:XXXX@localhost/roundcube';
```

**Comment by trapperhoney** [comment]
Bah, I think I found the problem :\  I saw a message like this in the error log:
```
strlen(): Passing null to parameter #1 ($string) of type string is deprecated in /var/www/roundcubemail-1.5.3/program/actions/mail/index.php on line 181
```

It looks like something was still referencing the 1.5.3 version of this file.  Force-restarting php8.1-fpm.service fixed the problem.  So it looks like the old version of some of the php files were cached by the webserver's php service?

Sorry for the noise, and thanks for your help!
